src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { Children, ComponentPropsWithoutRef, ReactElement, ReactNode, cloneElement } from 'react';
+import styled from 'styled-components';
+
+import { ChevronRightIcon } from '../../icons';
+import { Text } from '../Text/Text';
+
+import { Colors } from '../../essentials';
+import { theme } from '../../essentials/theme';
+import { get } from '../../utils/themeGet';
+import { Box } from '../Box/Box';
+
+interface InvertedStyle {
+    /**
+     * Adjust color for display on a dark background
+     * @default false
+     */
+    inverted?: boolean;
+}
+
+interface BreadcrumbsProps extends InvertedStyle {
+    /**
+     * Content of the Breadcrumbs
+     * @required
+     */
+    children: ReactNode;
+}
+
+interface LinkProps extends ComponentPropsWithoutRef<'a'>, InvertedStyle {}
+
+type ItemProps = InvertedStyle;
+
+const BreadcrumbsList = styled.ul`
+    padding: 0;
+    list-style: none;
+    display: flex;
+`;
+
+const BreadcrumbsListItem = styled.li`
+    display: flex;
+`;
+
+const Breadcrumbs = ({ children, inverted }: BreadcrumbsProps): JSX.Element => {
+    const arrayChildren = Children.toArray(children);
+
+    return (
```

Comment:
It is recommended to have the breadcrumb wrapped with the `nav` tag for a11y. 
I found a very good example on the [a11y style guide website](https://a11y-style-guide.com/style-guide/section-navigation.html#kssref-navigation-breadcrumbs) we can use as a markup example.

- [ ] follow the a11y recommendations